utopia-remix/app/util/use-sort-compare-project.tsx

```diff
@@ -57,7 +57,7 @@ export function useProjectIsOnActiveOperation() {
           case 'private':
           case 'sharing':
             return op.type === 'delete' && op.projectId === project.proj_id
-          case 'trash':
+          case 'archive':
```

Comment:
it'd be nice to make this type safe with a guard

Reply:
interestingly, it threw an error but wasn't caught in the CI
<img width="1004" alt="image" src="https://github.com/concrete-utopia/utopia/assets/7003853/6ba9a929-14d0-4de1-b2cd-3c39a1cf0676">

Reply:
o.O